_pytest/fixtures.py

```diff
@@ -1068,6 +1068,9 @@ def parsefactories(self, node_or_obj, nodeid=NOTSET, unittest=False):
         self._holderobjseen.add(holderobj)
         autousenames = []
         for name in dir(holderobj):
+            # Skip properties before the following getattr executes them.
+            if isinstance(type(holderobj).__dict__.get(name), property):
```

Comment:
Just an idea: what if instead of explicitly skipping properties we just make sure we only accept function objects? After all fixtures can only be created using functions.